common/src/main/java/org/apache/falcon/security/SecurityUtil.java

```diff
@@ -115,6 +115,16 @@ public static boolean isAuthorizationEnabled() {
                 "falcon.security.authorization.enabled", "false"));
     }
 
+    /**
+     * Checks if CSRF filter is enabled in the configuration.
+     *
+     * @return true if falcon.security.csrf.enabled is enabled, false otherwise
+     */
+    public static boolean isCSRFFilterEnabled() {
```

Comment:
In the UI, using default configurations, authentication doesn't happen and you can enter any username and password and emulate that user. In that case one doesn't need to do csrf, right?